crates/pallet-bioauth/src/lib.rs

```diff
@@ -400,6 +415,13 @@ pub mod pallet {
                     }
                     true
                 });
+                if removed {
+                    // Emit an event.
+                    Self::deposit_event(Event::AuthenticationRemoved {
+                        validator_public_key: public_key.clone(),
+                        reason,
+                    });
+                }
```

Comment:
This most likely should not be in the `ActiveAuthentications::mutate` block

Reply:
Thanks, fixed - https://github.com/humanode-network/humanode/pull/649/commits/dc6e74201f31c98ec7abb9869754f3f9fc00fac0